compose-app/build.gradle.kts

```diff
@@ -33,6 +33,10 @@ android {
     kotlinCompilerExtensionVersion = Versions.compose
   }
 
+  kotlinOptions {
+    freeCompilerArgs += "-Xexplicit-api=strict"
```

Comment:
what does this flag do ?

Reply:
it enforces us to explicitly specify the access modifier and returned type, so it's easier to identify unintentional exposed APIs.